### DISCUSSION TITLE
[Bug] Conversation mode listens to the model's answer and considers it part of my response

### DISCUSSION THREAD

**Discussion by orenyomtov** [open-webui/open-webui#4152] [General]
# Bug Report

## Description
When having a voice conversation, it listens to the TTS answer of the model and acts as if I said it.
For example here I only said "Six" in my response:
<img width="714" alt="image" src="https://github.com/user-attachments/assets/8c508520-5726-4fcf-8f17-82b9e16bdbd1">

**Steps to Reproduce:**
Click the earphones logo and have a short conversation, then look at the chat's transcript.

**Expected Behavior:**
Your side will only say what you've said

**Actual Behavior:**
Your side will also have the previous model's answer.

## Environment

- **Open WebUI Version:** v0.3.10
- **Ollama (if applicable):** v0.3.0

- **Operating System:** macOS
- **Browser (if applicable):** Chrome

**Comment by ALLMI78** [comment]
hi guys, i also had this problem and found a solution:

Settings > Voice > Allow Voice Interruption in Call >>> set this to **OFF** if enabled

in my case that was the reason for the feedback loop
